config/checkstyle_non_main_files_checks.xml

```diff
@@ -24,6 +24,15 @@
     <message key="regexp.filename.mismatch"
       value="CI folder should only contain executables files"/>
   </module>
+  <module name="RegexpOnFilename">
+    <property name="id" value="regexpOnGithubWorkflows"/>
+    <property name="folderPattern" value="[\\/].github[\\/]workflows([\\/]|$)"/>
+    <property name="fileNamePattern" value="^([a-z][a-z0-9]*)(-[a-z0-9]+)*\..*$"/>
```

Comment:
We shouldn't end the regular expression with `.*`. You are allowing non-kebab-case characters in, and even if it isn't the real extension.
Example: `kebab.NoTMyKebab_CASE.yml`

I recommend `[a-z]+` as file extensions already covers the extensions we will be matching against.

Reply:
I agree, done.